src/test/regress/expected/isolation_global_pid.out

```diff
@@ -0,0 +1,155 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-start-session-level-connection s1-worker-begin s1-worker-select s2-coordinator-citus_dist_stat_activity s2-coordinator-citus_worker_stat_activity s2-coordinator-get_all_active_transactions s2-coordinator-get_global_active_transactions s1-worker-commit s1-stop-session-level-connection
+create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-start-session-level-connection:
+    SELECT start_session_level_connection_to_node('localhost', 57637);
+
+start_session_level_connection_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-worker-begin:
+    SELECT run_commands_on_session_level_connection_to_node('BEGIN');
+
+run_commands_on_session_level_connection_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+step s1-worker-select:
+    SELECT run_commands_on_session_level_connection_to_node('SET citus.force_max_query_parallelization TO ON; SELECT * FROM dist_table');
```

Comment:
SET citus.enable_local_execution TO false until we fix: https://github.com/citusdata/citus/issues/5282